assets/src/edit-story/components/library/panes/pageLayouts/pageLayout.js

```diff
@@ -17,24 +17,141 @@
+  position: absolute;
+  z-index: -1;
+  background-color: ${({ theme }) => theme.colors.loading.primary};
+  border-radius: ${({ theme }) => theme.border.radius.default};
+  overflow: hidden;
+`;
+PreviewPageWrapper.propTypes = {
+  pageSize: PageSizePropType.isRequired,
+};
+
+const HoverControls = styled.div`
+  height: ${({ pageSize }) => pageSize.containerHeight}px;
+  width: ${({ pageSize }) => pageSize.width}px;
+  position: absolute;
+  opacity: ${({ isActive }) => (isActive ? 1 : 0)};
+  cursor: pointer;
+  border-radius: ${({ theme }) => theme.border.radius.default};
+  overflow: hidden;
 
-const PageLayoutPlaceholder = styled.div`
-  height: ${PAGE_LAYOUT_PANE_WIDTH / PAGE_RATIO}px;
-  width: ${PAGE_LAYOUT_PANE_WIDTH}px;
-  background-color: ${({ theme }) => theme.colors.bg.white};
+  ${KEYBOARD_USER_SELECTOR} &:focus {
+    outline: ${({ theme }) => `2px solid ${theme.colors.selection} !important`};
+  }
 `;
+HoverControls.propTypes = {
+  pageSize: PageSizePropType.isRequired,
+  isActive: PropTypes.bool.isRequired,
+};
+
+const PageLayoutTitle = styled.div`
+  position: absolute;
+  bottom: 0;
+  background-color: ${({ theme }) => theme.colors.grayout};
+  padding: 8px;
+  font-size: 12px;
+  line-height: 22px;
+  width: 100%;
+`;
+
+function PageLayout(props) {
+  const { page, pageSize, onConfirm, requiresConfirmation } = props;
 
-function PageLayout() {
-  return <PageLayoutPlaceholder />;
+  const [isActive, setIsActive] = useState(false);
+  const [isConfirming, setIsConfirming] = useState(false);
+  const containElem = useRef(null);
+
+  useFocusOut(containElem, () => setIsActive(false), []);
+
+  const handleClick = useCallback(() => {
+    if (requiresConfirmation) {
+      setIsConfirming(true);
+    } else {
+      onConfirm();
+    }
+  }, [setIsConfirming, onConfirm, requiresConfirmation]);
+
+  const handleCloseDialog = useCallback(() => {
+    setIsConfirming(false);
+  }, [setIsConfirming]);
+
+  const handleConfirmDialog = useCallback(() => {
+    onConfirm();
+    setIsConfirming(false);
+  }, [onConfirm, setIsConfirming]);
+
+  return (
+    <>
+      <PageLayoutWrapper pageSize={pageSize} role="listitem">
+        <PreviewPageWrapper pageSize={pageSize}>
+          <PreviewErrorBoundary>
+            <PreviewPage
+              pageSize={pageSize}
+              page={page}
+              animationState={
+                isActive
+                  ? STORY_ANIMATION_STATE.PLAYING
+                  : STORY_ANIMATION_STATE.RESET
+              }
+            />
+          </PreviewErrorBoundary>
+        </PreviewPageWrapper>
+        <HoverControls
+          ref={containElem}
+          pageSize={pageSize}
+          onFocus={() => setIsActive(true)}
+          onMouseEnter={() => setIsActive(true)}
+          onMouseLeave={() => setIsActive(false)}
+          onClick={handleClick}
+          isActive={isActive}
+          aria-label={page.title}
+          tabIndex="0"
```

Comment:
The tabbing works great but I think a <kbd>Enter</kbd> to apply it would be good to add.